### ISSUE TITLE
[A11y] Topic Info Tab

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#2634]
## Current Output

https://user-images.githubusercontent.com/9396084/107034739-24564900-67dd-11eb-9d87-b671a28ed6f5.mp4

## Issues Identified
The talkback reads this screen as **Oppia** at start, instead it should be **Topic: Fractions** (Assigned to @Sparsh1212)

Note: To understand the above issue(s) completely, it is recommended that you setup Talkback and play with the app keeping it on and that will give you better context.

**Accessibility Guide**: https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide

**Comment by mschanteltc** [comment]
The "See More" button should still be available to learners, even though talkback reads the entire topic description when it's collapsed. This can help users who have mobility issues but not visibility issues, so those who want to read the full description (instead of having it momentarily on the talkback reader).

**Comment by Sparsh1212** [comment]
@rt4914 I would love to work on the first part of this issue. If possible, then please assign this to me.

**Comment by rt4914** [comment]
> The "See More" button should still be available to learners, even though talkback reads the entire topic description when it's collapsed. This can help users who have mobility issues but not visibility issues, so those who want to read the full description (instead of having it momentarily on the talkback reader).

Sound correct. Updated the description.
